source/slang/slang-language-server-ast-lookup.cpp

```diff
@@ -451,6 +451,13 @@ struct ASTLookupStmtVisitor : public StmtVisitor<ASTLookupStmtVisitor, bool>
         return false;
     }
 
+    bool visitLabelStmt(LabelStmt* stmt)
+    {
+        if (_isLocInRange(context, stmt->label.loc, stmt->label.getContent().getLength()))
```

Comment:
Does this imply that preprocessor usage for the label, or doing a `#include` that holds the label won't work?

Both seem sort of iffy so perhaps that's ok. It might be a little surprising tho, with preprocessor/`#includes` notionally producing the same thing as having a source file contained the expanded output.